### ISSUE TITLE
Hide usernames and hide timestamps options don't work

### ISSUE THREAD

**Issue by rekamath2** [akhanubis/anu_twitch_chat_overlay#62]
Hide usernames and hide timestamps options don't work, both still appear when both options are enabled

**Comment by AirCordRicky** [comment]
Yep, both timestamps and username hide option do not function as of my comment. Hope you have the time at some point to fix that, but otherwise very cool extension!
